Gruntfile.coffee

```diff
@@ -117,4 +107,4 @@ This file is generated by `grunt build`, do not edit it by hand.
     json2 = pkg.jqueryJSON
     json1[key] = json2[key] for key of json2
     json1.author.name = pkg.author
-    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2))
+    grunt.file.write('chosen.jquery.json', JSON.stringify(json1, null, 2) + "\n")
```

Comment:
@pfiller it won't remove it from the list AFAIK (they were parsing this file for tags). Thus, it would mean that the list is not really read-only :smile:.
And even if it removes it, I don't think we care. The jQuery plugin registry never got the traction intended (probably because it waited mor than 1 year before replacing the previous registry so everyone found another solution)